helm/kubernetes-cluster-autoscaler-chart/values.yaml

```diff
@@ -10,3 +10,8 @@ image:
 
 cluster:
   id: "test-cluster"
+
+ports:
+  metrics:
+    name: metrics
+    port: 8085
```

Comment:
If you change the port will it be also changed in the autoscaler or just in the service?

Reply:
Just in the service. I'm unsure if it makes sense to adapt the CA parameter to actually change the port. Reason is that this port doesn't need to be exposed via service in order to be available for prometheus but only because having service requires some port. Then again, the port is hard coded in `prometheus-config-controller` which also requires a service object to be present in order to be able to find CA POD. This solution is a bit ugly, but I'm not starting to refactor `prometheus-config-controller` at this point because there exists already a story where future prometheus architecture is evaluated and it's not said that this config controller will be part of it.

Reply:
Ack. Changed.